### PR TITLE
5831 | Create new behavior plugin

### PR DESCRIPTION
**Pull request by doctorgde1**
https://redmine.devbranch.work/issues/5831

### CODE REVIEW

File: web/modules/custom/paragraph_behavior/src/Plugin/paragraphs/Behavior/ImageBehavior.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\paragraphs\Entity\ParagraphsType;
+use Drupal\paragraphs\ParagraphInterface;
+use Drupal\paragraphs\ParagraphsBehaviorBase;
+
+/**
+ * @file
+ * Additional behavior for paragraph "Photo with text" type.
+ */
```
Comment (Y0r) [inline web/modules/custom/paragraph_behavior/src/Plugin/paragraphs/Behavior/ImageBehavior.php:16]: useless file description

classes with namespaces doesn't need file description

File: web/modules/custom/paragraph_behavior/src/Plugin/paragraphs/Behavior/ImageBehavior.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+   */
+  public static function isApplicable(ParagraphsType $paragraphs_type) {
+    return $paragraphs_type->id() == "photo_with_text";
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function view(array &$build, Paragraph $paragraphs_entity, EntityViewDisplayInterface $display, $view_mode): void {
+    $image_position = $paragraphs_entity->getBehaviorSetting($this->getPluginId(), 'image_position');
```
Comment (Y0r) [inline web/modules/custom/paragraph_behavior/src/Plugin/paragraphs/Behavior/ImageBehavior.php:43]: missing 3 parameter for default value

**Review by Y0r — CHANGES_REQUESTED** [review]

File: web/modules/custom/paragraph_behavior/src/Plugin/paragraphs/Behavior/ImageBehavior.php (hunk 2 above)
Comment (doctorgde1) [reply web/modules/custom/paragraph_behavior/src/Plugin/paragraphs/Behavior/ImageBehavior.php:43]: Do i need change it like this ?

`... getBehaviorSetting($this->getPluginId(), 'image_position'**, 'left'**);`

Comment (doctorgde1) [reply web/modules/custom/paragraph_behavior/src/Plugin/paragraphs/Behavior/ImageBehavior.php:43]: I am not sure, dose 3 means three or third

**Review by Y0r — APPROVED** [review]

**Pull request merged**